modules/appeals_api/spec/workers/notice_of_disagreement_clean_up_week_old_pii_spec.rb

```diff
@@ -4,27 +4,14 @@
 
 describe AppealsApi::NoticeOfDisagreementCleanUpWeekOldPii, type: :job do
   describe '#perform' do
-    it 'removes PII for NoticeOfDisagreements that reached a completed status a week ago' do
-      create :notice_of_disagreement, :completed_a_week_ago
-      expect(AppealsApi::NoticeOfDisagreement.has_pii).not_to be_empty
-      described_class.new.perform
-      expect(AppealsApi::NoticeOfDisagreement.has_pii).to be_empty
-    end
+    it 'calls AppealsApi::RemovePii' do
+      service = instance_double(AppealsApi::RemovePii)
+      allow(AppealsApi::RemovePii).to receive(:new).and_return(service)
+      allow(service).to receive(:run!)
 
-    it 'does not log to sentry if there was nothing to work on' do
-      expect(AppealsApi::NoticeOfDisagreement.has_pii).to be_empty
-      worker = described_class.new
-      expect(worker).not_to receive(:log_message_to_sentry)
-      worker.perform
-    end
+      described_class.new.perform
 
-    it 'logs to sentry if no pii was removed /but/ some should have been' do
-      allow_any_instance_of(described_class).to(
-        receive(:no_notice_of_disagreements_are_ready_to_have_pii_expunged?).and_return(false)
-      )
-      worker = described_class.new
-      expect(worker).to receive(:log_message_to_sentry)
```

Comment:
Are we concerned with testing our logging? 
I just ran into a spot where we weren't testing our logging, and it turns out that the logging wasn't actually working, which could have been caught with a test. Thoughts?

Reply:
I don't disagree with testing the logging in principle - although in this case, we're testing that the call to the Logging "class" or wrapper exists, which is done. Testing the logging functionality here would be out of scope.

Reply:
K. I need to understand this more!